gui/gui/inc/LinkDef.h

```diff
@@ -1,3 +1,290 @@
-#include "LinkDef1.h"
-#include "LinkDef2.h"
-#include "LinkDef3.h"
```

Comment:
I think that was needed on Windows, due to some string/buffer/file limitations...

Reply:
Also with cling?

Reply:
Not sure... We'll see

Reply:
As one can see from file history, before three separate GUI dictionaries were build.
And then in 2013 they were merged in one - while cling does not like to have many dictionaries for same library.

Reply:
I think it was a source file (dictionary) size limitation with Visual Studio. Let's see if it's still the case...

Reply:
But CLING generates one dictionary file anyway.

Reply:
Well, if it compile AND works, then fine with me